src/app/models/submission.server.model.ts

```diff
@@ -115,6 +81,17 @@ const emailSubmissionSchema = new Schema<IEmailSubmissionSchema>({
   },
 })
 
+/**
+ * Returns an object which represents the encrypted submission
+ * which will be posted to the webhook URL.
+ */
+
```

Comment:
nit: Remove this newline and update the comment to be more descriptive. Maybe something about why null is returned here.

Reply:
will do